workers/unity/Packages/io.improbable.gdk.core/Tests/Editmode/Connection/ReceptionistFlowTests.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.IO;
+using Improbable.Gdk.TestUtils.Editor;
+using Improbable.Gdk.Tools;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.Core.EditmodeTests.Connection
+{
+    [TestFixture]
+    public class ReceptionistFlowTests
+    {
+        private SpatialDeploymentManager deploymentManager;
+        private Improbable.Worker.CInterop.Connection connection;
```

Comment:
Why is this full typed?

Reply:
It conflicts with the current namespace: `namespace Improbable.Gdk.Core.EditmodeTests.Connection
`